src/style.css

```
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --color: #fff;
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.search {
+  position: relative;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: inherit;
+  box-sizing: border-box;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding: 0 0 0 62px;
+  background: var(--color);
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background: var(--color);
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, var(--color) 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.icon {
+  position: absolute;
+  top: 0;
+  left: 26px;
+}
+
+.input.input--small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  padding: 0 0 0 33px;
+}
+
+.small_input:hover {
+  background: var(--color);
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small_input:focus {
+  background: linear-gradient(180deg, var(--color) 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search_small {
+  position: relative;
+}
+
+.icon.icon--small {
```

```suggestion
.icon {
```
Do not increase specificity